servers/physics_3d/godot_area_3d.cpp

```diff
@@ -341,6 +346,8 @@ void GodotArea3D::compute_gravity(const Vector3 &p_position, Vector3 &r_gravity)
 		} else {
 			r_gravity = v.normalized() * get_gravity();
 		}
+	} else if (gravity_apply_transform) {
+		r_gravity = get_transform().get_basis().xform(get_gravity_vector()) * get_gravity();
```

Comment:
With this code, point gravity will always take the transform into account, but directional gravity will only take the transform into account when `gravity_apply_transform` is true. This seems like an odd inconsistency. However, making it consistent would require breaking behavior compatibility, which is what PR #90166 does.

Reply:
I'd say there's a valid use case for not applying transform to the direction, much less so for the point, if we *are* to break compatibility I'd argue it would be to make *both* controllable, not make the directional one local